docs/changehistory/2.5.0.md

```diff
@@ -117,18 +121,17 @@ IModelApp now supports passing FrontendSecurityOptions at startup. In particular
 
 The @bentley/build-tools package now supports ESLint ^6.8.0 and a configuration file to match, the now deprecated, TSLint configuration. The TSLint configuration and custom rules will still exist for the rest of iModel.js 2.x and will be removed in iModel.js 3.0.
 
-To start using the new configuration add the following to the package.json to extend the build-tools package and run the eslint rules against all `ts` and `tsx` files: 
+To start using the new configuration add the following to the package.json to extend the build-tools package and run the eslint rules against all `ts` and `tsx` files:
 
 ```json
 "scripts": {
   "lint": "eslint ./src/**/*.{ts,tsx} 1>&2",
 },
 "devDependencies": {
-  "eslint": "^6.8.0",
+  "eslint": "^7.10.0",

```

Comment:
I'd leave this as is and just note the updates in the `NextVersion.md`.

Reply:
oops, that was find&replace :)